arclightning_web/settings.html

```diff
@@ -0,0 +1,93 @@
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<!doctype html>
+<html>
+
+<head>
+    <link href="https://fonts.googleapis.com/css?family=Do+Hyeon" rel="stylesheet">
+    <!--bootstrap-->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u"
+        crossorigin="anonymous">
+    <!-- Vuejs -->
+    <script src="https://cdn.jsdelivr.net/npm/vue"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+    <style>
+        .centerItem {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+        }
+
+        .formattedText {
+            font-family: 'Do Hyeon', sans-serif;
+            font-size: 42px;
+        }
+    </style>
+</head>
+
+<body>
+
+
+<nav class="navbar navbar-light bg-light">
+  <a class="navbar-brand" href="#">
+    <img src="../arclightning_frontend/images/logo.png" style="height: 250%;" alt="">
+  </a>
+	<ul class="nav nav-tabs navbar-right">
+		<li role="presentation" class="active" style="font-family: 'Do Hyeon', sans-serif; font-size: 18px;"><a href="#">settings</a></li>
+		<li role="presentation"  style="font-family: 'Do Hyeon', sans-serif; font-size: 18px;color: black;"><a href="login.html">log out</a></li>
+	</ul>
+
+</nav> 
+    <div class="row" style="width:100vw; height:100vh; padding:0; margin:0;">
+        <div id="settings" class="col-xs-12 col-lg-8 col-lg-offset-2" style="height:100vh; width: 100%;">
+
+            <form id="settingForm" class="formattedText" v-on:submit="submitSettings()">
+               <br> <input type="text" maxlength="200" id="name" style=" top: 10%; text-align:center;width: 100%; border:none; border-bottom: 2px solid black;" v-model="name" v-bind:style="{color: profile_fgcolor, 'background-color': profile_bgcolor}"required /><br>
+
+		<h1 class="formattedText centerItem" >Name color:</h1>
+                <input class="centerItem" type="color" style="height: 50px; width:80%;" id="profile_fgcolor"  value="#333999" v-model="profile_fgcolor"/>
+
+		<h1 class="formattedText" style="display:inline">Background color:</h1>
+                <input class="centerItem" type="color" style="height: 50px; width: 80%;" id="profile_bgcolor" value="#219444" v-model="profile_bgcolor" />
+
+                <p class="formattedText">Volume: </p>
+		<input class="centerItem" type="range" id="volume" style="width: 80%;" min="0.0" max="1.0" step = "0.05" v-model="volume"/>
```

Comment:
can we have this be one of those slider bars that fill up vs a dot on a line?

like this 
▮▮▮▮▮▮▯▯▯▯

Reply:
So, I looked into this and I thiiiink we'd have to use an external library to make it available to all browsers. Do you want me to send you some of the libraries or leave it as is?